### ISSUE TITLE
Does not support subquery

### ISSUE THREAD

**Issue by henrya2** [dbis-uibk/relax#198]
```SQL
SELECT d FROM S WHERE S.b = (SELECT b FROM R WHERE R.a=3);
```

I use the test UIBK R, S, T dataset.

**Comment by henrya2** [comment]
WITH statement is not supported either.

**Comment by rlaiola** [comment]
> ```sql
> SELECT d FROM S WHERE S.b = (SELECT b FROM R WHERE R.a=3);
> ```
> 
> I use the test UIBK R, S, T dataset.

Indeed, subqueries are not supported (as indicated [here](https://dbis-uibk.github.io/relax/help#sql-where)).

**Comment by rlaiola** [comment]
> WITH statement is not supported either.

As for the WITH clause you can find a working example [here](https://github.com/dbis-uibk/relax/issues/195).
